src/types.ts

```diff
@@ -1,5 +1,9 @@
 import { TransportMode, TransportSubmode, StopPlace, Quay } from '@entur/sdk'
 
+export interface GenericKeyValueObject {
```

Comment:
Føler egentlig dette er unødvendig. Bruk heller `Record<string, any>` der det er behov for denne.

Reply:
Det var gjort for å slippe så mange `// eslint-disable-next-line @typescript-eslint/no-explicit-any`, men da bytter jeg til `Record<string, any>` og legger på eslint-disable i stedet:)